src/pyscheduling/FS/FmCmax.py

```diff
@@ -3,31 +3,25 @@
 from typing import ClassVar, List
 
 import pyscheduling.FS.FlowShop as FlowShop
-import pyscheduling.FS.FS_methods as FS_methods
+from pyscheduling.FS.solvers import BIBA
 import pyscheduling.Problem as Problem
 from pyscheduling.FS.FlowShop import Constraints
 from pyscheduling.Problem import Job, Objective
+from pyscheduling.core.base_solvers.base_solver import BaseSolver
```

Comment:
Can be shortened from `import bases_solvers.base_solver.BaseSolver` to `import base_solvers.BaseSolver`